herbert_core/utilities/strings/str_length.m

```diff
@@ -0,0 +1,40 @@
+function L = str_length (str)
+% Lengths of strings
+%
+%   >> L = str_length (str)
+%
+% Performs the same function as Matlab intrinsic function strlength, except that
+% it also works for Matlab versions prior to R2017a i.e. before the new string
```

Comment:
Its because I have some other bits of code independent of Horace that use these utilities but which I want to run on older versions of Matlab. In the spirit of 'do no harm' I wanted to allow this to be used until the overhaul of #1485 is implemented.